src/main/java/wolox/training/model/User.java

```diff
@@ -64,6 +67,14 @@ public void setUsername(String username) {
             "Illegal Argument, username cannot be NULL.");
     }
 
+    public String getPassword() {
```

Comment:
Password would be serialized when serializing the user. Check and find out how to tell this model that the password should only be "writtable".

Reply:
Ok! :+1: I set the password as only writtable when using JSON